docs/hardhat-network/README.md

```diff
@@ -1,41 +1,44 @@
-# Hardhat Network
+# Hardhat Network: An Explanation <!--theoretical, not practical; for studying, not for working-->
```

Comment:
Do we need the `: An Explanation` part? I would assume that this section explains what Hardhat network is.

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1657/commits/07fecf1d7361787ef2b15f1a3ddface15935791b